### PR TITLE
GRN ngraph op revision

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
GRN ngraph op revision:
- [x] FusedOp inheritance removal
- [x] type_prop tests update
- [x] Attribute tests update

### Tickets:
 - 44630 (45773, 53942)

### CODE REVIEW

File: ngraph/core/src/op/grn.cpp [hunk 1]
```diff
@@ -22,29 +17,25 @@ using namespace ngraph;
 
 NGRAPH_SUPPRESS_DEPRECATED_START
```
Comment (ilya-lavrenov) [inline ngraph/core/src/op/grn.cpp:18]: we can remove `NGRAPH_SUPPRESS_DEPRECATED_START` as well

File: ngraph/core/include/ngraph/op/grn.hpp [hunk 2]
```diff
@@ -7,9 +7,7 @@
 #include <memory>
 
 #include "ngraph/node.hpp"
-#include "ngraph/op/util/fused_op.hpp"
-
-NGRAPH_SUPPRESS_DEPRECATED_START
+#include "ngraph/op/op.hpp"
```
Comment (jdanieck) [inline ngraph/core/include/ngraph/op/grn.hpp:10]: nit: I think `#include "ngraph/node.hpp"` is not needed anymore.

**Review by jdanieck — APPROVED** [review]

File: ngraph/core/src/op/grn.cpp (hunk 1 above)
Comment (mitruska) [reply ngraph/core/src/op/grn.cpp]: Of course, done. Thanks!

File: ngraph/core/include/ngraph/op/grn.hpp (hunk 2 above)
Comment (mitruska) [reply ngraph/core/include/ngraph/op/grn.hpp:9]: Ok, removed

**Review by ilyachur — APPROVED** [review]

**Pull request merged**